templates/terraform/resource.erb

```diff
@@ -382,6 +380,7 @@ func resource<%= resource_name -%>Update(d *schema.ResourceData, meta interface{
     }
 
     log.Printf("[DEBUG] Updating <%= object.name -%> %q: %#v", d.Id(), obj)
+<%= lines(compile('templates/terraform/pre_update/update_mask.erb')) if object.all_user_properties.select { |v| v.is_a?(Api::Type::UpdateMask) }.first -%>
```

Comment:
Ideal to move it - `pre_update` is where we store custom code, which this no longer is.